src/aiidalab_qe/plugins/xps/setting.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+"""Panel for XPS plugin.
+
+"""
+import ipywidgets as ipw
+import traitlets as tl
+from aiida.orm import Group, QueryBuilder, StructureData
+
+from aiidalab_qe.common.panel import Panel
+
+base_url = "https://github.com/superstar54/xps-data/raw/main/pseudo_demo/"
+
+
+def install_pseudos(pseudo_group="pseudo_demo_pbe"):
+    import os
+    from pathlib import Path
+    from subprocess import run
+
+    url = base_url + pseudo_group + ".aiida"
+
+    env = os.environ.copy()
+    env["PATH"] = f"{env['PATH']}:{Path.home().joinpath('.local', 'bin')}"
+
+    def run_(*args, **kwargs):
+        return run(*args, env=env, capture_output=True, check=True, **kwargs)
+
+    run_(["verdi", "archive", "import", url])
+
+
+class Setting(Panel):
+    title = "XPS Settings"
+    identifier = "xps"
+    input_structure = tl.Instance(StructureData, allow_none=True)
+    protocol = tl.Unicode(allow_none=True)
+
+    core_hole_treatment_title = ipw.HTML(
+        """<div style="padding-top: 0px; padding-bottom: 0px">
+        <h4>Core hole treatment</h4></div>"""
+    )
+    core_hole_treatment_help = ipw.HTML(
+        """<div style="line-height: 140%; padding-top: 0px; padding-bottom: 5px">
+        You have three options:<br>
+        (1) XCH(smear): places the excited electron into the conduction band, suitable for extend system.<br>
+        (2) XCH(fixed): places the excited electron into the conduction band, suitable for extend system.<br>
+        (3) Full: remove one electron from the system, suitable for molecule. </div>"""
+    )
+
+    pseudo_title = ipw.HTML(
+        """<div style="padding-top: 0px; padding-bottom: 0px">
+        <h4>Pseudopotential</h4></div>"""
```

Comment:
Just to make things easier to read:
```suggestion
        <h4>Core-Hole pseudopotential group</h4></div>"""
```

Reply:
I changed as you suggested.